media/js/portal/blackrock_webcam.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable security/detect-non-literal-fs-filename */
```

Comment:
@nikolas I tried to resolve most of the eslint issues.  The two remaining issues I'm not sure how to resolve.  It complains that `draw_image_flow` is defined but not used; though it's actually called on the page.  It also complains that `ImageFlow` is undefined, but like before, I suspect that its in the global scope of the page.

Reply:
The second error is pretty common when running eslint on older javascript that relies on global variables. We've had to do this a lot in Mediathread, you can just add `/* global ImageFlow */` to the top of the file to tell eslint that it should be available globally.

And yeah.. it looks like `draw_image_flow()` is used in a template file. I'll probably just add some type of eslint-ignore for this specific line.